### ISSUE TITLE
<Thumos experiment> the dropout layer in train.prototxt was cancelled ,but i found overfitting when trying to test my caffemodel by using eval_detection.py

### ISSUE THREAD

**Issue by fanxx17** [VisionLearningGroup/R-C3D#63]
the dropout layer in train.prototxt was cancelled ,but i found overfitting when trying to test my caffemodel by using eval_detection.py:
however,the caffemodel provided in Thumos's README.md has beautiful performance. Does train.prototxt  really nead dropout layer when you train Thumos? thank you for your reply
...
}
#layer {
#  name: "drop6"
#  type: "Dropout"
#  bottom: "fc6"
#  top: "fc6"
#  dropout_param {
#    dropout_ratio: 0.5
#  }
#}
...
